### ISSUE TITLE
Fix RCE bug

### ISSUE THREAD

**Issue by ui0ppk** [ui0ppk/roblox-master#2]
Would be thankful ngl.

**Comment by imaloser71** [comment]
no

**Comment by samdisk11** [comment]
WHO ARE YOU

**Comment by ui0ppk** [comment]
> WHO ARE YOU

I'm David Hasselhoff

**Comment by samdisk11** [comment]
sus

**Comment by 3UR** [comment]
no leave it !

**Comment by ui0ppk** [comment]
> no leave it !

shut up "La" troll

lalalallalaalalalalalalaal

**Comment by b2vm** [comment]
i think you're gonna attempt on making a revival by yourself
integrating slikenet is incredibly easy so why make this issue

https://github.com/SLikeSoft/SLikeNet

**Comment by b2vm** [comment]
Kay and Why and the S variety

**Comment by RoverUser** [comment]
what rce

**Comment by KLanausse** [comment]
https://github.com/latte-soft/0x1D/issues/1#issuecomment-1292951016

**Comment by ui0ppk** [comment]
> [latte-soft/0x1D#1 (comment)](https://github.com/latte-soft/0x1D/issues/1#issuecomment-1292951016)

That's for studio, the RCE is on RCCService RAKNet

**Comment by KLanausse** [comment]
I ask that you disregard my previous comment as looking at it now it kind of stupid.

The previous comment was a link to a reply to a different RCE which read

> RCE Deez Nuts

**Comment by ui0ppk** [comment]
> I ask that you disregard my previous comment as looking at it now it kind of stupid.
> 
> The previous comment was a link to a reply to a different RCE which read
> 
> > RCE deez nuts.

Though I didn't see that. My bad.

**Comment by b2vm** [comment]
You will never be a real Roblox 2016 Source Code git.rip

**Comment by ui0ppk** [comment]
> You will never be a real Roblox 2016 Source Code git.rip

Oh so you wanna challenge me?
I can re-upload the unmodified version.

**Comment by Tarvey** [comment]
> > You will never be a real Roblox 2016 Source Code git.rip
> 
> Oh so you wanna challenge me? I can re-upload the unmodified version.

do it but in a different repo

**Comment by iRevolutionDev** [comment]
> > You will never be a real Roblox 2016 Source Code git.rip
> 
> Oh so you wanna challenge me? I can re-upload the unmodified version.

Upload the original source to another branch or repository

**Comment by b2vm** [comment]
> > You will never be a real Roblox 2016 Source Code git.rip
> 
> Oh so you wanna challenge me? I can re-upload the unmodified version.

Si

**Comment by b2vm** [comment]
That would be thankful ngl 😀(You will never be a real RCCService)

**Comment by ui0ppk** [comment]
> That would be thankful ngl 😀(You will never be a real RCCService)

By the way, if you see ANYONE saying the unmodified source code has no RCE, they are lying.

**Comment by ui0ppk** [comment]
https://github.com/ui0ppk/roblox-master-unmodified
@b2vm @iRevolutionDev @Tarvey

**Comment by ui0ppk** [comment]
Yes, I did just fork it, because it would of been a waste of time to reupload this to my account.

**Comment by b2vm** [comment]
> > That would be thankful ngl 😀(You will never be a real RCCService)
> 
> By the way, if you see ANYONE saying the unmodified source code has no RCE, they are lying.

No

**Comment by Afliex** [comment]
> https://github.com/ui0ppk/roblox-master-unmodified @b2vm @iRevolutionDev @Tarvey

yo where did the sdk link on mega go 😭

**Comment by CosmoBoy67** [comment]
No don't leave it!
